### DISCUSSION TITLE
Implementation of getListingOffers in Products API

### DISCUSSION THREAD

**Discussion by philipherrmann** [saleweaver/python-amazon-sp-api#117] [Feature requests]
Hi all!
I would like to consume the getListingOffers Operation (cf. https://github.com/amzn/selling-partner-api-docs/blob/b39997cfaed2c00557d875481c76401d2fcfa7c3/references/product-pricing-api/productPricingV0.md#get-productspricingv0listingssellerskuoffers ) and it should probably be a method of the api.Products class. Would you welcome a PR for that? If anyone else is quicker than me, I would of course appreciate this feature becoming available without contributing it myself :)

Best regards
Philip

**Comment by tentuku** [comment]
Hi,
I am very grateful to all involved for the addition of the getListingOffers Operation.
I've been trying to consume the getItemOffers Operation for a while now, but I can't get it to work.
I know getItemOffers Operation is the same Products class, but can anyone suggest a way to use it?
